bot/meeting.py

```diff
@@ -1,23 +1,52 @@
+import logging
 from datetime import datetime
-from aiogram import Bot
+
+from aiogram.utils.i18n import gettext as _
 from apscheduler.schedulers.asyncio import AsyncIOScheduler
-from .custom_types import SendMessage, ChatId
+
 from .constants import day_of_week, jobstore
+from .custom_types import ChatId, SendMessage
 from .messages import make_daily_messages
-from .state import load_state, ChatState
-import logging
-from aiogram.utils.i18n import gettext as _
+from .state import load_state, save_state, get_joined_users
 
 
 async def send_meeting_messages(chat_id: ChatId, send_message: SendMessage):
     chat_state = await load_state(chat_id=chat_id)
-    await send_message(chat_id=chat_id, message=_("Meeting time!"))
-    if not chat_state.joined_users:
-        await send_message(chat_id=chat_id, message=_("Nobody has joined the meeting!"))
+    topic_id = chat_state.topic_id
+    current_day = datetime.now().weekday()
+    await send_message(chat_id=chat_id, message=_("Meeting time!"), message_thread_id=topic_id)
+     
+    joined_users = await get_joined_users(chat_state)
+    today_workers = [user for user in joined_users if current_day in user.meeting_days]
+    if not today_workers:
+        await send_message(chat_id=chat_id, message=_("Nobody has joined the meeting!"), message_thread_id=topic_id)
     else:
-        for username in chat_state.joined_users:
-            for message in make_daily_messages(username=username):
-                await send_message(chat_id=chat_id, message=message)
+
+        # Creating list of joined to meeting users
+        today_usernames = []
+        for user in today_workers:
+            today_usernames.append(f"@{user.username}")
+
+        # Getting daily messages
+        daily_messages = make_daily_messages(usernames=" ".join(today_usernames))
```

Comment:
@belyakova-anna this feature requires a scenario proposal because it's necessary to show how users reply to such messages.